kazoo/tests/test_cache.py

```diff
@@ -51,6 +92,27 @@ def spy_client(self, method_name):
         method = getattr(self.client, method_name)
         return patch.object(self.client, method_name, wraps=method)
 
+    def _gc_wait(self):
+        if self.handler.name == 'sequential_threading_handler':
+            while not self.client.handler.completion_queue.empty():
+                self.client.handler.sleep_func(0.1)
+        else:
+            # Let hub of gevent/eventlet be in a short-time loop
```

Comment:
Should we check here that the queue is empty?

Reply:
Modified in 6c0920c